SPECS/python-cpuinfo/python-cpuinfo.spec

```diff
@@ -0,0 +1,223 @@
+%global srcname cpuinfo
+%global sum Getting CPU info
+
+Summary:        %{sum}
+Name:           python-%{srcname}
+Version:        7.0.0
+Release:        4%{?dist}
+License:        MIT
+URL:            https://github.com/workhorsy/py-cpuinfo
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        https://files.pythonhosted.org/packages/source/p/py-%{srcname}/py-%{srcname}-%{version}.tar.gz
+# s390x support
+Patch0:         py-cpuinfo-s390x.patch
+BuildArch:      noarch
+# https://github.com/workhorsy/py-cpuinfo/issues/55
+# ExclusiveArch:  %%{ix86} x86_64 %%{power64} s390x noarch
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-xml
+%if %{with_check}
+BuildRequires:  python3-pip
+BuildRequires:  python3-atomicwrites
```

Comment:
same comment as for python-conda-packagehandling

Reply:
done:)